extensions/panache/mongodb-panache/runtime/src/main/java/io/quarkus/mongodb/panache/reactive/runtime/ReactiveMongoOperations.java

```diff
@@ -217,11 +216,11 @@ public static ReactiveMongoDatabase mongoDatabase(Class<?> entityClass) {
     }
 
     private static Uni<Void> persist(ReactiveMongoCollection collection, Object entity) {
```

Comment:
Maybe we should also break reactive mongo panache compatibility, by returning a Uni<InsertOneResult> here?
But we should at least try to be consistent between reactive mongo and reactive panache mongo.

Reply:
@postremus we try to avoid returning mongodb specific objects inside Panache. 
Update and delete didn't return UpdateResult nor DeleteResult so I prefer to keep void here.

Reply:
Not sure if it's possible, but maybe we could return the updated entity?